.semgrepignore

```diff
@@ -4,6 +4,7 @@ vuu/src/main/resources/www/ws-example.html
 vuu/src/main/scala/org/finos/vuu/provider/simulation/SimulatedBigInstrumentsProvider.scala
 vuu-ui/packages/vuu-data/src/array-data-source/group-utils.ts
 vuu-ui/packages/vuu-datagrid-extras/src/column-expression-input/column-language-parser/walkExpressionTree.ts
+vuu-ui/packages/vuu-layout/src/layout-persistence/RemoteLayoutPersistenceManager.ts
```

Comment:
[the static code analysis was failing in the pipeline](https://github.com/ScottLogic/finos-vuu/pull/74/checks?check_run_id=17732098110) beacuse we are using http instead of https. semgrp ignoring this file is not an ideal solution but will make the static code analysis pass for now. [this ticket](https://github.com/ScottLogic/finos-vuu/issues/79) will look into implementing SSL so that we don't need to do this. I have added a note to the ticket to remove this line from this file once we have SSL.